newsAnalysis/data/collect.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+'''
+Handles transport of big data for analysis
+'''
+from data.model import Article
+import unicodecsv as csv
+import os
+
+
+def download_articles_to_file(session, file_path, source_id=1):
+    '''
+    connection
+        A psycopg2 connection
+    file_path : String
```

Comment:
where is the psycopg2 connection? Do we still need that module?

Reply:
I need to update that documentation to say that is a SqlAlchemy connection